cardinal/ecs/store/iface.go

```diff
@@ -0,0 +1,61 @@
+package store
+
+import (
+	"encoding/json"
+
+	"pkg.world.dev/world-engine/cardinal/ecs/archetype"
+	"pkg.world.dev/world-engine/cardinal/ecs/component"
+	"pkg.world.dev/world-engine/cardinal/ecs/entity"
+	"pkg.world.dev/world-engine/cardinal/ecs/filter"
+	ecslog "pkg.world.dev/world-engine/cardinal/ecs/log"
+	"pkg.world.dev/world-engine/cardinal/ecs/storage"
+)
+
+type IReader interface {
```

Comment:
can we just call these `Reader`, `Writer`? and then as you said change the `IManager` to `Manager` in the next PR

Reply:
done